integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/setRemovePropertiesWithNashornJsTestProxy.xml

```diff
@@ -0,0 +1,83 @@
+    <target>
+        <inSequence>
+            <script language="nashornJs"><![CDATA[
+          var payload = mc.getPayloadXML();
+          var expression = "//*[local-name()='Code']";
+          var xpath = mc.getXpathResult(expression);
+          var nodeList = xpath.selectNodes(payload);
+          var symbol = nodeList.get(0).getText();
+          mc.setPayloadXML(
+              "<m:getQuote xmlns:m=\"http://services.samples\">"
+                + "<m:request>"
+                    + "<m:symbol>"+ symbol +"</m:symbol>"
+                + "</m:request>"
+            + "</m:getQuote>");
+          mc.setProperty("axis2Property", "AXIS2_PROPERTY","axis2");
+          mc.setProperty("transportProperty", "TRANSPORT_PROPERTY","transport");
+          mc.setProperty("operationProperty", "OPERATION_PROPERTY","operation");
+
+	 ]]></script>
+            <log level="custom">
+                <property expression="get-property('axis2', 'axis2Property')" name="Axis2_Property"/>
+                <property expression="get-property('transport', 'transportProperty')" name="Transport_Property"/>
+                <property expression="get-property('operation', 'operationProperty')" name="Operation_Property"/>
+            </log>
+            <send>
+                <endpoint>
+                    <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
+                </endpoint>
+            </send>
+        </inSequence>
+        <outSequence>
+            <script language="nashornJs"><![CDATA[
+
+          var payload = mc.getPayloadXML();
+
+          var expression = "//*[local-name()='symbol']";
+          var xpath = mc.getXpathResult(expression);
+          var nodeList = xpath.selectNodes(payload);
+          var symbol = nodeList.get(0).getText();
+
+          expression = "//*[local-name()='last']";
+          xpath = mc.getXpathResult(expression);
+          nodeList = xpath.selectNodes(payload);
+          var price = nodeList.get(0).getText();
+
+          mc.setPayloadXML(
+              "<m:CheckPriceResponse xmlns:m=\"http://services.samples/xsd\">"
+            + "<m:Code>"+ symbol +"</m:Code>"
+            + "<m:Price>"+ price +"</m:Price>"
+            + "</m:CheckPriceResponse>");
+          mc.removeProperty("axis2Property","axis2");
+          mc.removeProperty("operationProperty","operation");
+          mc.removeProperty("transportProperty","transport");
+
+	 ]]></script>
+            <log level="custom">
+                <property expression="get-property('axis2', 'axis2Property')" name="Axis2_Property_After_Remove"/>
+                <property expression="get-property('transport', 'transportProperty')" name="Transport_Property_After_Remove"/>
+                <property expression="get-property('operation', 'operationProperty')" name="Operation_Property_After_Remove"/>
+            </log>
+            <send/>
+        </outSequence>
+    </target>
+</proxy>
```

Comment:
Better to have a new line.

Reply:
Hi Sumedha,
Fixed with following PR #1511